src/vector/_compute/lorentz/boost_beta3.py

```diff
@@ -60,29 +60,14 @@ def cartesian_t(lib, x1, y1, z1, t1, betax, betay, betaz):
     ty = yt
     tz = zt
     tt = gamma
-    return transform4D.cartesian_t(
-        lib,
-        xx,
-        xy,
-        xz,
-        xt,
-        yx,
-        yy,
-        yz,
-        yt,
-        zx,
-        zy,
-        zz,
-        zt,
-        tx,
-        ty,
-        tz,
-        tt,
-        x1,
-        y1,
-        z1,
-        t1,
-    )
+
+    row_x = (xx, xy, xz, xt)
+    row_y = (yx, yy, yz, yt)
+    row_z = (zx, zy, zz, zt)
+    row_t = (tx, ty, tz, tt)
+    row_1 = (x1, y1, z1, t1)
+
+    return transform4D.cartesian_t(lib, *row_x, *row_y, *row_z, *row_t, *row_1)
```

Comment:
@jpivarski This creates a tuple, which is not in the allowed uncompyle features. Should we allow tuple, or just use `# fmt: off` / `# fmt: on` around the call?

Reply:
I've commented on that below. It's not uncompyle6; the "What is this?" error message was written by me—the idea is that the feature hasn't been whitelisted so we should decide if we want to allow it and if so, let it through the sieve.

I doubt it would be a problem for any backend except Numba, and with Numba it might make for slower code or it might not. As you point out, LLVM might optimize it away. But the motivation for it—to make the words line up in the source code—is not strong enough to bother with all of that. If we just want to format this nicely, we should use those formatting pragmas.

Reply:
Using fmt comments to keep this nicely formatted without tuples.